packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -174,6 +182,12 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
                   }
                 }}
                 onKeyDown={(e) => {
+                  if (e.ctrlKey && e.key === 'd') {
+                    e.preventDefault();
+                    e.stopPropagation();
+                    onOpenSidebar?.();
+                    return;
+                  }
```

Comment:
You don't need this here because the click handler in the translation view will take care of this

Reply:
What do you mean "the click handler in the translation view"? Do you mean the keyboard shortcut handler?

Reply:
Done